tsconfig.json

```
@@ -6,5 +6,5 @@
       "~*": ["./*"]
     }
   },
-  "include": ["src/**/*"]
+  "include": ["src/**/*", "__tests__/sum.test.ts"]
```

I generally prefer the approach of keeping tests along side the thing they're testing.
i.e. `sum.test.ts` should be in the same folder as `sum.ts`.

---

```suggestion
  "include": ["src/**/*"]
```
I think you no longer need this now?